index.ts

```diff
@@ -1100,6 +1512,30 @@ export async function endpoint({ path, query, headers, method, body }) {
           .commentCreated.$emit({ comment });
         await repo.commentCreated.$emit({ comment });
       }
+      if (event.action === "created" && event.comment) {
+        // const comment = repo.issues
+        //   .one({ number: event.issue.number })
+        //   .comments.one({ id: event.comment.id });
+        // await repo.issues
+        //   .one({ number: event.issue.number })
+        //   .commentCreated.$emit({ comment });
+        console.log("org comment")
+        // console.log(org.name)
+        await org.commentCreated.$emit({ org });
```

Comment:
Ah, this gives me more context on your question from yesterday. Ideally you could pass the `event.comment` to directly as the arg to`.$emit` because that should be a `Comment`:
```ts
await org.commentCreated.$emit({ comment: event.comment });
```
Does that not work?

Reply:
I don't think that it did.